mods/ra/maps/allies-13/allies13.lua

```diff
@@ -0,0 +1,301 @@
+		if actor.Owner == Greece then
+			Trigger.RemoveProximityTrigger(id)
+			Greece.MarkCompletedObjective(TakeMoney)
+		end
+	end)
+
+	local flamers = Utils.Where(Map.ActorsInWorld, function(self) return self.Owner == USSR and self.Type == "ftur" end)
+	Utils.Do(flamers, function(actor)
+		Trigger.OnDamaged(actor, function(building)
+			if building.Health < building.MaxHealth * 9/10 then
+				building.StartBuildingRepairs()
+			end
+		end)
+	end)
+
+	Trigger.OnTimerExpired(function()
+		DateTime.TimeLimit = 0
+		Trigger.AfterDelay(1, function() UserInterface.SetMissionText("We're too late!", USSR.Color) end)
+		USSR.MarkCompletedObjective(HoldOut)
+	end)
+end
+
+PlaceCharges = function()
+	for generatorID = 1, 8 do
+		Trigger.OnEnteredProximityTrigger(Generators[generatorID].CenterPosition, WDist.FromCells(1), function(actor, id)
+			if actor.Type == "e6" then
+				Trigger.RemoveProximityTrigger(id)
+				ChargePlaced[generatorID] = true
+				Actor.Create("flare", true, { Owner = Greece, Location = Generators[generatorID].Location + CVec.New(0,-1) })
+				Media.PlaySpeechNotification(Greece, "ExplosiveChargePlaced")
+				Media.DisplayMessage(UserInterface.Translate("explosive-charge-placed"), UserInterface.Translate("engineer"))
+			end
+		end)
+	end
+end
+
+FlameTowerTriggers = function()
+	for flameID = 1, 8 do
+		Trigger.OnEnteredProximityTrigger(FlameConsoles[flameID].CenterPosition, WDist.FromCells(1), function(actor, id)
+			if actor.Type == "e6" then
+				Trigger.RemoveProximityTrigger(id)
+				if not FlameTowers[flameID].IsDead then
+					Media.DisplayMessage(UserInterface.Translate("flame-turret-deactivated"), UserInterface.Translate("console"))
+					FlameTowers[flameID].Kill()
+				end
+			end
+		end)
+	end
+
+	Trigger.OnEnteredProximityTrigger(TurncoatConsole.CenterPosition, WDist.FromCells(1), function(actor, id)
+		if actor.Type == "e6" then
+			Trigger.RemoveProximityTrigger(id)
+			Actor.Create("ftur", true, { Owner = Turkey, Location = TurncoatFlameTurret.Location })
+		end
+	end)
+
+	Trigger.OnEnteredProximityTrigger(TwoTowerConsole.CenterPosition, WDist.FromCells(1), function(actor, id)
+		if actor.Type == "e6" then
+			Trigger.RemoveProximityTrigger(id)
+
+			Media.DisplayMessage(UserInterface.Translate("flame-turret-deactivated"), UserInterface.Translate("console"))
+			Utils.Do(DualTowerActors, function(actor)
+				if not actor.IsDead then
+					actor.Kill()
+				end
+			end)
+		end
+	end)
+end
+
+CameraTriggers = function()
+	local heavyCamTriggered
+	Trigger.OnEnteredFootprint(HeavyTankCamFootprint, function(actor, id)
+		if actor.Owner == Greece and not heavyCamTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			heavyCamTriggered = true
+
+			local heavyCam = Actor.Create("camera", true, { Owner = Greece, Location = HeavyTankCam.Location })
+			Media.DisplayMessage(UserInterface.Translate("old-flametowers"), UserInterface.Translate("engineer"))
+			Utils.Do(TankRifles, function(a)
+				IdleHunt(a)
+			end)
```

Comment:
```suggestion
			Utils.Do(TankRifles, IdleHunt)
```
Since I blabbed about DoomPatrol, I'll put this here too.